index.js

```diff
@@ -30,7 +30,8 @@ module.exports = connection => {
   const settings = {
     client: 'pg',
     useNullAsDefault: true,
-    connection
+    connection,
+    ...knexSnakeCaseMappers()
```

Comment:
It would be good if we could use the knexfile here, rather than duplicating settings in 2 places

Reply:
Separate ticket? :D

Reply:
nah, it would take longer to write the ticket than to just make the change. But I see what you're saying, stay in your lane 🚗